scripts/test/Muon/grouping_tab/grouping_table_presenter_test.py

```diff
@@ -370,6 +370,16 @@ def test_updating_range_min_to_be_greater_than_range_max_displays_warning_and_vi
         self.view.warning_popup.assert_called_with('Maximum of group asymmetry range must be greater than minimum')
         self.assertEqual(self.view.warning_popup.call_count, 2)
 
+    def test_that_valid_period_strings_assessed_as_valid(self): 
+        self.data_context.num_periods = self._fake_num_periods
+
+        valid = self.presenter.validate_periods('84447-9')
+
+        self.assertTrue(valid>0)
```

Comment:
can we have a better assert?

Reply:
This test set was actually incomplete, have gone back and added a couple more tests in and improved this one.